lib/chef/knife/winrm.rb

```diff
@@ -29,6 +29,13 @@ class Winrm < Knife
 
       include Chef::Knife::WinrmCommandSharedFunctions     
 
+      FAILED_BASIC_HINT ||= "Hint: Please check winrm configuration 'winrm get winrm/config/service' AllowUnencrypted flag on remote server."
+      FAILED_NOT_BASIC_HINT ||= <<-eos.gsub /^\s+/, ""
```

Comment:
I personally prefer the `gsub` (though I'm not crazy about it) over losing the indentation.

Any one else have opinions here?